main.go

```diff
@@ -283,7 +285,7 @@ func main() {
 		tm.Register("eth", api.NewBlockAPI(logsdb, cfg.Chainid, pl, cfg))
 		tm.Register("eth", api.NewGasAPI(logsdb, cfg.Chainid, pl, cfg))
 	}
-	if hasTx && hasBlocks && hasLogs && hasMempool {
+	if hasTx && hasBlocks && hasLogs {
```

Comment:
Do tests pass with this? I'm suspicious that some of the API calls will generate errors because they expect to be able to query the mempool table and it doesn't exist.